n3/n3-tests.ts

```diff
@@ -0,0 +1,204 @@
+/// <reference path="./index.d.ts" />
+/// <reference path="../node/index.d.ts" />
+
+
+/**
+  [?] has correct naming convention
+      I am not sure what the correct naming convention is.
+      Is it n3.d.ts or index.d.ts?
+
+  checked compilation succeeds  
+   [+] tsc --noImplicitAny --target es5 ./index.d.ts
+   [x] tsc --noImplicitAny --target es6 ./index.d.ts
+
+  [x] has a test file with the suffix of -tests.ts
+
+  checked the test file
+   [x] tsc --noImplicitAny --target es5 --module commonjs ./n3-tests.ts 
+   [x] tsc --noImplicitAny --target es6 --module commonjs ./n3-tests.ts 
+
+*/
+
+import * as N3 from "n3";
+import * as fs from "fs";
+import * as stream from "stream";
+
+function test_serialize() {
+
+    var writer: N3.N3Writer = N3.Writer(
+        {
+            format: "ttl",
+            prefixes: {
+                foaf: "http://xmlns.com/foaf/0.1",
```

Comment:
The type you're testing here is `{ [key: string]: string; }`, so no need to be so detailed.